bin/build-application-solvcon-devenv.sh

```diff
@@ -1,5 +1,6 @@
 # SCDE: SOLVCON Dev Env
```

Comment:
Why is this file in `$root/bin/` instead of `$root/applications/solvcon/`?

Reply:
`bin/` directory implies `executable` under `bin/` is very likely an entry point of a program. Currently `devenv` and `build-application-solvcon-devenv.sh` are orthogonal. I would like to make people feel like there are two entry points (for different purposes and tools). When features like `devenv launch <application>` https://github.com/solvcon/devenv/discussions/23 is delivered, we won't need `build-application-solvcon-devenv.sh` anymore.

Both `$root/bin/` and `$root/applications/solvcon/` works for me. Let me know any of them works better for you. I am happy to move the file.

Reply:
I think the application directory is better for the application specific executable.  It doesn't need to be fixed right away.  How about creating an issue tracking the movement?

Reply:
Certainly. It is https://github.com/solvcon/devenv/issues/26